### ISSUE TITLE
Register flag macro is not in the core

### ISSUE THREAD

**Issue by roigcarlo** [KratosMultiphysics/Kratos#812]
The flag related macros are not defined in the code. This is linked with #811

It appears to be defined in:
- `add_c2c_variables_to_python.cpp`
- `add_containers_to_python.cpp`
- `add_dem_variables_to_python.cpp`
- `add_deprecated_variables_to_python.cpp`

In my opinion. it should be in `defines.h`

The block of code I am speaking about is: 

```c++
#ifdef KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION
#undef KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION
#endif
#define KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION(flag) \
 scope().attr(#flag) = boost::ref(flag)      \
 
#ifdef KRATOS_REGISTER_IN_PYTHON_FLAG
#undef KRATOS_REGISTER_IN_PYTHON_FLAG
#endif
#define KRATOS_REGISTER_IN_PYTHON_FLAG(flag) \
    KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION(flag);   \
    KRATOS_REGISTER_IN_PYTHON_FLAG_IMPLEMENTATION(NOT_##flag)
```

**Comment by pooyan-dadvand** [comment]
I would add it to the core

**Comment by roigcarlo** [comment]
Fixed in #815
